test/support/factory.ts

```diff
@@ -16,7 +17,7 @@ import { sequence } from "./helpers";
 export const gFlags = (attrs: Partial<BFlags> = {}): BFlags => {
   return {
     "service-token": "valid-token",
-
+    "api-origin": undefined,
```

Comment:
No, but rather because typescript was complaining about that.

This function returns a `BFlags`:
```
type BFlags = {
    "service-token": string;
    "api-origin": string | undefined;
} & {
    json: boolean | undefined;
}
```